instrumentation/google.golang.org/grpc/otelgrpc/interceptor_test.go

```diff
@@ -381,6 +382,54 @@ func TestStreamClientInterceptor(t *testing.T) {
 	_ = streamClient.CloseSend()
 }
 
+// TestStreamClientInterceptorWithError tests a situation that streamer returns an error.
+func TestStreamClientInterceptorWithError(t *testing.T) {
```

Comment:
Please use [goleak](https://github.com/uber-go/goleak) to verifying that here no leaks